### PR TITLE
update changelog for v0.9.0, v0.8.2, v0.8.3

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

Would appreciate another sentence or two on prometheus metrics for the highlights section if anyone more involved has an idea.

Feel free to suggest edits to the notes, or if you think any particular item should be re-categorized.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+  * Show a better error message when trying to get an incomplete restore's logs (#496, @nrb)
+  * Stop processing when setting a backup deletion request's phase to `Deleting` fails (#500, @nrb)
+  * Add library code to install Ark's server components (#437 #506, @marpaia)
+  * Properly handle errors when backing up additional items (#512, @carlpett)
+  * Run post hooks even if backup actions fail (#514, @carlpett)
+  * GCP: fail backup if upload to object storage fails (#510, @nrb)
+  * AWS: don't require `region` as part of backup storage provider config (#455, @skriss)
+  * Ignore terminating resources while doing a backup (#526, @yastij)
+  * Log to stdout instead of stderr (#553, @ncdc)
+  * Move sample minio deployment's config to an emptyDir (#566, @Thomas Runyon)
```
Comment (ncdc) [inline CHANGELOG.md]: fix user id?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+#### [v0.9.0](https://github.com/heptio/ark/releases/tag/v0.9.0) - 2018-07-05
+
+##### Highlights:
+  * Ark now has support for backing up and restoring Kubernetes volumes using a free open-source backup tool called [restic](https://github.com/restic/restic).
+    This provides users an out-of-the-box solution for backing up and restoring almost any type of Kubernetes volume, whether or not it has snapshot support
+    integrated with Ark. For more information, see the [documentation](https://github.com/heptio/ark/blob/master/docs/restic.md).
+  * Support for Prometheus metrics has been added! Get more visibility into the behavior of Ark.
```
Comment (nrb) [inline CHANGELOG.md]: Maybe something like: "View total number of backup attempts, including success or failure, total back up size in bytes, and backup durations with more metrics to come"

Wording could probably use work, but those are the metrics that'll be making the cut.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+#### [v0.9.0](https://github.com/heptio/ark/releases/tag/v0.9.0) - 2018-07-05
```
Comment (ncdc) [inline CHANGELOG.md]: Since we missed 7/5 let's update to 7/6

Comment (skriss) [reply CHANGELOG.md]: will do

**Comment by Bradamant3** [comment]
LGTM

**Pull request merged**